AAI/AAIConnectProfile.md

```diff
@@ -286,9 +285,7 @@ the Broker.
         1.  Access tokens for GA4GH use MUST be a [GA4GH JWT](#ga4gh-jwt-format) using
             [Passport-Scoped Access Token format](#passport-scoped-access-token-issued-by-broker).
 
-        2.  Access tokens do not contain GA4GH Claims directly in the access token.
-
-        3.  Access tokens MAY contain non-GA4GH Claims directly in the access token.
+        2.  Access tokens MAY contain additional non-GA4GH Claims directly in the access token.
```

Comment:
Doesn't this subtely change the meaning by allowing a passport scoped access token to be a 'fat' token with all the visa content (whereas this was explicitly disallowed before)?